File: src/main/java/com/handwoong/everyonewaiter/service/StoreService.java
package com.handwoong.everyonewaiter.service;

import com.handwoong.everyonewaiter.dto.store.StoreReqDto;
import com.handwoong.everyonewaiter.dto.store.StoreResDto;
import java.util.List;

public interface StoreService {

    Long register(String username, StoreReqDto storeDto);

    StoreResDto findStore(String username, Long storeId);

    List<StoreResDto> findStoreList(String username);
}

File: src/main/kotlin/com/handwoong/everyonewaiter/service/store/StoreService.kt
package com.handwoong.everyonewaiter.service.store

import com.handwoong.everyonewaiter.dto.store.StoreRequest
import com.handwoong.everyonewaiter.dto.store.StoreResponse

interface StoreService {

    fun register(username: String, storeDto: StoreRequest)

    fun update(username: String, storeId: Long, storeDto: StoreRequest)

    fun delete(username: String, storeId: Long)

    fun findStore(username: String, storeId: Long): StoreResponse

    fun findStoreById(storeId: Long): StoreResponse

    fun findStoreList(username: String): List<StoreResponse>

}
